### DISCUSSION TITLE
Conditional key bindings based on window title

### DISCUSSION THREAD

**Discussion by stiang** [ghostty-org/ghostty#3705] [Q&A] [answered]
Does Ghostty support something like Kitty’s [`map --when-focus-on title:tmux [...]`](https://sw.kovidgoyal.net/kitty/mapping/#conditional-mappings-depending-on-the-state-of-the-focused-window)?

This might not be useful to a whole lot of people, and I almost wish I hadn’t started using it to make switching between terminal emulators easier, but I did and now it’s a bit of a blocker.

**Comment by pluiedev** [comment, marked as answer]
Hyperspecific conditional behavior belongs to #2353
